roles/ffh.mesh_tunneldigger/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: Install tunneldigger
+  include_tasks: install_tunneldigger.yml
+  when: domains is defined and sn is defined
```

Comment:
regarding all `when: domains is defined and sn is defined`, what is it you want to achieve?

Reply:
It should not do anything otherwise/skip the role.